controllers/usersController.js

```diff
@@ -1,3 +1,38 @@
+const { User } = require('../services/authService');
+const { oneTimePass } = require('../services/registerService');
+
 const getUsers = async (req, res) => res.sendStatus(200);
 
-module.exports = { getUsers };
+async function handleGetUsers(req, res) {
+  const showAllUser = await User.find({});
+  return res.send(JSON.stringify(showAllUser));
+}
+
+async function handleGetUser(req, res) {
+  const user = await User.retrieve({ email: req.params.email });
+  return res.send(JSON.stringify(user));
+}
+
+async function handleUpdateUser(req, res) {
+  try {
+    console.log(req.body);
+    await User.update({ email: req.body.email }, { data: req.body });
+    return res.status(200).json({ message: 'user update' });
+  } catch (err) {
+    return res.status(401).json({ message: err.message });
+  }
+}
+
+async function handleDeleteUser(req, res) {
+  try {
+    await User.find({ email: req.body.email });
+    await oneTimePass.delete({ email: req.body.email });
+    return res.status(200).json({ message: 'The user has been deleted' });
```

Comment:
If the user is with a free plan, you should also delete the account

Reply:
1. why delete the account? 
2. why not just set the account as `disabled`/ `closed`?